src/libraries/Microsoft.PowerFx.Core/Public/DisplayNames/DisplayNameProvider.cs

```diff
@@ -10,7 +10,7 @@
 
 namespace Microsoft.PowerFx.Core
 {
-    internal abstract class DisplayNameProvider
+    public abstract class DisplayNameProvider
     {
         public abstract bool TryGetLogicalName(DName displayName, out DName logicalName);
```

Comment:
Suggest to make methods internal to allow refactoring later.

Reply:
agreed